lib/fs.js

```diff
@@ -259,13 +259,13 @@ function exists(path, callback) {
   maybeCallback(callback);
 
   function suppressedCallback(err) {
-    callback(err ? false : true);
+    callback(!err);
   }
 
   try {
     fs.access(path, F_OK, suppressedCallback);
   } catch {
-    return callback(false);
```

Comment:
Either here I see no reason to remove `return`.
In this case if there is no `return` keyword, then function keep executing with callback called.